_posts/2018-11-06-october-updates.md

```diff
@@ -0,0 +1,61 @@
+---
+title: October Updates - Hacktoberfest, new releases and more. 
+author: Hong Phuc Dang 
+date: 2018-11-06
+banner:
+  image: octupdates.jpg
+---
+# This is a recap of open source activities and development at Zalando in the month of October.
+
+ 
+## Project Highlights 
+
+- [**Connexion version 2.0**](https://github.com/zalando/connexion) with OpenAPI 3 support is ready, [check out what is new in our latest release!](https://opensource.zalando.com/blog/2018/11/connexion-20-release/) Connexion is the [Swagger/OpenAPI](https://swagger.io) first framework for Python on top of Flask with automatic endpoint validation & OAuth2 support. With 87 active contributors and more than 1,000 repositories that depend on Connexion worldwide makes this project one of the most successful open source releases of Zalando. 
+
+- [**Postgres-Operator**](https://github.com/zalando-incubator/postgres-operator) after one year of development, this operator now manages more than 500 Postgres clusters across a large number of Kubernetes installations inside Zalando. Our engineers do not need manually configure numerous Kubernetes objects, they just submit a single text file describing the desired Postgres cluster, and the operator takes care of the rest. Postgres-operator was first started by Zalando’s database team to provide a managed PostgreSQL service for Kubernetes. [Try out the operator here!](https://github.com/zalando-incubator/postgres-operator) 
+
+- [**Flair**](https://github.com/zalandoresearch/flair) Zalando recently released a new version of this open source Natural Language Processing framework, it now runs on both Linux and Mac, [click here to test!](https://github.com/zalandoresearch/flair). The project was initiated by [Zalando Research](https://research.zalando.com/). Flair gives users the ability to tag, classify and understand the meanings of email messages, customer responses, website comments, or any other scenario where users submit text feedback to be automatically classified or otherwise processed.
+
+
+## Inside Zalando Open Source 
+- [**Zalando hosted a Hack Night at the Berlin Tech Hub to celebrate Hacktoberfest - the month of open source.**](https://github.com/zalando) The main event started off with a number of lightning talks by open source projects, followed by a hacking session, where Zalando engineers gathered as teams and worked on challenges under domains of machine learning, database and web plug-ins. Project maintainers were present to support participants completing their very first contribution and pull request.
```

Comment:
The berlin office is not a tech hub :) it's just the Berlin office

Reply:
- "The main event started with..." Use of "started off" is somewhat antiquated these days.